### ISSUE TITLE
FEATURE - Lock Screen Widget

### ISSUE THREAD

**Issue by mikaelacaron** [mikaelacaron/brain-marks#155]
## Describe the feature you want the app to have that it currently doesn't
* Create a lock screen widget for viewing a specific category's tweets

## Describe alternatives you've considered
* None, I'm open to any other lock screen widget ideas, comment below!

## Additional context
* N/A

**Comment by SuzGupta** [comment]
I'm interested in trying this one as well. I want to put some more thought into what would be a truly useful lock screen widget from this app.

**Comment by mikaelacaron** [comment]
Yeah let me know! I'm only assigning one issue per person at the moment
Feel free to drop any ideas here and others may be interested as well!

**Comment by SuzGupta** [comment]
Yeah, I understand and that definitely seems like a fair approach, one issue at a time per contributor. I will comment further when I come up with some Lock Screen possibilities.

**Comment by SuzGupta** [comment]
Still pondering ideas, would still like to be assigned when/if current PR is accepted.

**Comment by mikaelacaron** [comment]
We could have a lock screen widget go directly to the `TweetList` view, and the user can choose which "list" meaning which category, they want a "hyperlink" for on their lock screen

The widget could be the icon that's saved for that category

**Comment by SuzGupta** [comment]
That sounds straighforward, what you propose above. Another option (that I suspect is a little harder to code, proposing this before trying) would be to show the last three saved Tweets and a tap would just open the app (so no deep link, just the default widget behavior). View would be kinda like this:
![BrainMarksLockscreenWidget](https://user-images.githubusercontent.com/5115383/197824338-dd646de7-60be-494f-bab9-385ef96bd751.PNG)

**Comment by mikaelacaron** [comment]
@SuzGupta It won't be more than say 3-5 words, so I would lean on not showing any individual tweets.
Cause Lock Screen widgets are supposed to be glanceable information

**Comment by SuzGupta** [comment]
Yeah, excellent point. So, I think this will be of type `.accessoryCircular` and I will have a go at implementing just what you suggested above 🤞
